class Error(Exception):
    pass


class Abort(Error):
    pass
